src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
Why would that be? I don't know what might return `null` in an Add action? I also wondered why `delete()` has `\Cake\Network\Response|null` as the baked action always returns a redirect.